tethys_apps/views.py

```diff
@@ -39,7 +40,7 @@ def library(request):
                 configured_apps.append(app)
             else:
                 unconfigured_apps.append(app)
-        elif user_can_access_app(request.user, app):
+        elif user_can_access_app(request.user, app) or not getattr(settings, "ENABLE_RESTRICTED_APP_ACCESS", False):
```

Comment:
The check on `ENABLE_RESTRICTED_APP_ACCESS` should be in `user_can_access_app`, no? Is there a reason we wouldn't put it in there?

Also, this only hides the app tile. What if the user enters the URL manually? Isn't there a middleware where we need to check for this?